common/serializers.py

```
@@ -0,0 +1,43 @@
+        # Instantiate the superclass normally
+        super().__init__(*args, **kwargs)
+
+        if fields is not None:
+            # Drop any fields that are not specified in the `fields` argument.
+            allowed = set(fields)
+            existing = set(self.fields)
+            for field_name in existing - allowed:
+                self.fields.pop(field_name)
+
+
+def create_validated_instance(serializer: Serializer, request: Request):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_validated_instance`

(ANN201)